Changelog.tex

```diff
@@ -0,0 +1,11 @@
+\section*{Changelog}
+The changelog lists the significant changes in the rules since the previous global RoboCup event. 
+Changes such as spelling and grammar fixes are not included. 
+\subsection*{Changes in 2018}
+
+\begin{itemize}
+	\item Introduction of changelog
+	\item \textbf{EE-GPSR} Simplify EE-GPSR: less categories, simpler scoring. Full detail in \href{https://github.com/RoboCupAtHome/RuleBook/pull/327}{PR 327}
+	\item \textbf{VizBox} Introduce VizBox to visualize challenge with \href{https://github.com/RoboCupAtHome/RuleBook/pull/367}{PR 367}
+	\item \textbf{DSPL Standard laptop} Mandatory use of official Standard Laptop for DSPL \href{https://github.com/RoboCupAtHome/RuleBook/pull/374}{PR 374}
+\end{itemize}
```

Comment:
Missing section for the list of buffed tests. I've in my ToDo list:
- Change Storing Groceries
  - Only moved objects score
  - PDF report for validation purposes
- Change Set the table & tidy up
  - Remove set the table
  - Tidy up part is a startup for the dishwasher test
  - Required by RoboCup federation as part as the P&G agreement